certbot/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
-*
+* certbot.\_internal.ocsp moved to public API, and is now available in package certbot.ocsp
```

Comment:
super nit: Rather than documenting the change to Certbot internals, I'd personally prefer we moved this to the "Added" section and say something like "Added certbot.ocsp to Certbot's API which can be used to determine the OCSP status of certificates."